numpy/core/numeric.py

```diff
@@ -2309,9 +2312,12 @@ def isclose(a, b, rtol=1.e-5, atol=1.e-8, equal_nan=False):
 
      absolute(`a` - `b`) <= (`atol` + `rtol` * absolute(`b`))
 
-    The above equation is not symmetric in `a` and `b`, so that
-    `isclose(a, b)` might be different from `isclose(b, a)` in
-    some rare cases.
+    Unlike the built-in `math.isclose`, the above equation is not symmetric
+    in `a` and `b` -- it assumes `b` is the reference value -- so that 
+    `isclose(a, b)` might be different from `isclose(b, a)`. Furthermore,
+    the default value of `atol` is not zero, and is targeted at numbers
+    of order unity; especially for numbers significantly smaller than one,
+    it can give false positives if not adjusted appropriately.
```

Comment:
The reason atol exists at all is that nothing is *relatively* close to zero. That should probably be in the docs here somewhere. Maybe:

"""
Furthermore, the default value of `atol` is not zero, and is used to determine what small values should be considered close to zero. The default value is appropriate for expected values of order unity: if the expected values are significantly smaller than one, it can result in false positives. `atol` should be carefully selected for the use case at hand. A zero value for atol will result in False if either `a` or `b` is zero.
"""

Reply:
That's probably acceptable. I don't think that this should be a crash-course on floating point math. I think it should be enough to get someone who is familiar with numerics to realize the implications of using this function.